pyproject.toml

```diff
@@ -29,7 +29,7 @@ include = [
 "Issues" = "https://github.com/requests-cache/requests-cache/issues"
 
 [tool.poetry.dependencies]
-python        = "^3.7"     # requests requires python >=3.7, <4.0
```

Comment:
The upper bound in `requests` was removed in https://github.com/psf/requests/pull/6333, and shipped with [2.29.0](https://github.com/psf/requests/releases/tag/v2.29.0)

Reply:
Good to know, thanks!